File: Jr/2022/CupcakeParty.py
# Owen Bélanger-Love

# Amount of regular boxes
regularBoxCount = int(input())

# Amount of small boxes
smallBoxCount = int(input())

# Calculate amount of cupcakes
cupcakeCount = 8 * regularBoxCount + 3 * smallBoxCount

# Amount of remaining cupcakes
# 28 students, each gets 1
leftover = max(cupcakeCount - 28, 0)

# Print amount of remaining cupcakes
print(leftover)

File: Jr/2022/CupcakeParty.java
// Owen Bélanger-Love

// Import Scanner class
import java.util.Scanner;

public class CupcakeParty {
    public static void main(String[] args) {
        // Create input Scanner
        Scanner input = new Scanner(System.in);

        // Amount of regular boxes
        int regularBoxCount = input.nextInt();

        // Amount of small boxes
        int smallBoxCount = input.nextInt();

        // Calculate amount of cupcakes
        int cupcakeCount = 8 * regularBoxCount + 3 * smallBoxCount;

        // Calculate remaining amount of cupcakes
        int leftover = cupcakeCount - 28;

        // No negative leftovers
        if (leftover < 0) {
            leftover = 0;
        }

        // Print amount of leftover cupcakes
        System.out.println(leftover);

        // Close input Scanner
        input.close();
    }
}
